features/xacml/org.wso2.carbon.identity.xacml.server.feature/resources/org.wso2.carbon.identity.xacml.server.feature.default.json

```diff
@@ -30,7 +30,7 @@
     "org.wso2.carbon.identity.entitlement.policy.publisher.CarbonBasicPolicyPublisherModule"
   ],
   "identity.entitlement.policy_point.pap.status_data_handlers": [
-    "org.wso2.carbon.identity.entitlement.SimplePAPStatusDataHandler"
+    "org.wso2.carbon.identity.entitlement.persistence.JDBCSimplePAPStatusDataHandler"
```

Comment:
Don't we need to set/remove`data_storage_type.xacml"` that we added with below PR?


https://github.com/wso2/carbon-identity-framework/pull/5866/files

Reply:
removed that line